src/ExampleData/focusedMapExampleData.js

```diff
@@ -0,0 +1,258 @@
+export const tileExampleData = [
```

Comment:
this element is exported twice - here as a regular export, and at the bottom as a default export.
it doesn't really matter which will remain, I prefer the regular export as it improves auto-completion.
If you're not sure how to fix this, or just want more info about exporting - talk to me, It just doesn't fit well in a comment.